### PR TITLE
Thing model support

### PR DESCRIPTION
**Pull request by lukesmolo**

### CODE REVIEW

File: packages/core/run.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#nodemon -w "./**" --ext "ts,json,jsonld" --exec "npm run build; npm run test test/ThingModelHelperTest.ts"
+nodemon -w "./**" --ext "ts,json,jsonld" --exec "npm run build; npm run test test/thing-model"
+#nodemon -w "./**" --ext "ts,json,jsonld" --exec "npm run build; npm run test test/thing-model/ThingModelHelperCompositionTest"
+#nodemon -w "./**" --ext "ts,json,jsonld" --exec "npm run build; npm run test test/thing-model/ThingModelHelperTest"
```
Comment (relu91) [inline packages/core/run.sh]: This File needs to be removed

File: packages/core/tsconfig.json [hunk 2]
```diff
@@ -5,6 +5,6 @@
         "outDir": "dist",
         "rootDir": "src"
     },
-    "include": ["src/**/*"],
+    "include": ["src/**/*", "src/tm-json-schema-validation.json"],
```
Comment (relu91) [inline packages/core/tsconfig.json:8]: @lukesmolo do you remember why you included this? shouldn't it be already included in `src/**/*` ?

**Review by relu91 — APPROVED** [review]
Seems good there are minor points that I'll cover, well done!

File: packages/core/run.sh (hunk 1 above)
Comment (lukesmolo) [reply packages/core/run.sh]: Yes!

**Pull request merged**